tests/numpy/test_gen_data.py

```diff
@@ -123,6 +128,24 @@ def test_does_not_flatten_arrays_of_lists(arr):
     assert isinstance(arr[0][0], list)
 
 
+@given(nps.arrays(object, 100, st.lists(max_size=0)))
+def test_generated_lists_are_distinct(ls):
```

Comment:
These three tests seem to be duplicated in `test_fill_values.py` - I assume they're only meant to be in one place?